src/style.css

```
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body * {
```

достатньо на body а решту тегів уже будуть наслідувати
```suggestion
body {
```